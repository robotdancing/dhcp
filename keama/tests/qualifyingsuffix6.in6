# ddns-domainname (aka qualifying-suffix)

# authoritative is mandatory
authoritative;

ddns-domainname ".biz";

# embedded
class "foo" {
    ddns-domainname ".bar";
}
