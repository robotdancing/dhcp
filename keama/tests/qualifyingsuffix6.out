{
  # ddns-domainname (aka qualifying-suffix)
  # authoritative is mandatory
  "Dhcp6": {
    "dhcp-ddns": {
      "enable-updates": false,
      "qualifying-suffix": ".biz"
    },
    "client-classes": [
      # embedded
      {
        "name": "foo"
//      /// Only global qualifying-suffix is supported
//      "qualifying-suffix": ".bar"
      }
    ]
  }
}
